database/data.sql

```diff
@@ -0,0 +1,66 @@
+-- Create the Tools Table
```

Comment:
May be nitpicking here, but maybe it would be a good idea to rename the file as "application" to specify that it deals with the "application" schema :)

Reply:
Done